src/components/refactor/dialogs/two-windings-transformer-creation/limits-pane/two-windings-transfomer-limits-pane-utils.js

```diff
@@ -0,0 +1,105 @@
+/**
```

Comment:
These 2 new files are strictly the same than in **line-creation/line-limits-pane** folder. I didn't expect that.
I think we have to use 2 common files for both line and 2wt.
Maybe in a new folder **dialog/limits-table** ?

your opinion ?

Reply:
After pondering it there's no real drawbacks to it, I've implemented it and taken care of applying the last changes you made to line limits pane

Reply:
yes, but it should be the same